lib/event.rb

```diff
@@ -0,0 +1,67 @@
+require "date"
+
+class Event
+  attr_reader :name, :food_trucks
+
+  def initialize(name)
+    @name = name
+    @food_trucks = []
+  end
+
+  def add_food_truck(food_truck)
+    @food_trucks << food_truck
+  end
+
+  def food_truck_names
+    @food_trucks.map(&:name)
+  end
+
+  def food_trucks_that_sell(item)
+    @food_trucks.filter do |food_truck|
+      food_truck.check_stock(item).positive?
+    end
+  end
+
+  def sorted_item_list
+    @food_trucks.map do |food_truck|
+      food_truck.inventory.keys.map(&:name)
+    end.flatten.uniq.sort
+  end
```

Comment:
Yup! Nice work. You could also use a flat_map on the outer-most enumerable and that would reduce one chained method from line 28, but not a big deal.